lib/bundler/cli/gem.rb

```diff
@@ -97,10 +98,14 @@ def run
             "test/test_helper.rb.tt" => "test/test_helper.rb",
             "test/newgem_test.rb.tt" => "test/#{namespaced_path}_test.rb"
           )
+        when "test-unit"
+          templates.merge!(
+            "test/test_newgem.rb.tt" => "test/test_#{namespaced_path}.rb"
```

Comment:
test-unit can use `XXX_test.rb` style.
So we can use `XXX_test.rb` here like other test framework.